packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -0,0 +1,171 @@
+        Retrieve an existing instance by its name.
+
+        Return None if it does not exist.
+        """
+        try:
+            res = self.req.get(f'{self.baseurl}/v1/instances')
+            data = res.json()
+            res.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise CorelliumApiException(data.get('error', str(e)))
+
+        for instance in instances:
```

Ruff (0.8.2)

`instances` may be undefined, or defined from star imports

(F405)